free-3.md

```diff
@@ -10,41 +10,91 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Some people use white noise to help them concentrate or fall asleep.
+Some people also have preferences based on the frequency spectrum used
+to generate a noise.
+For example, some prefer pink noise to white noise.
+The DSL would let people customize random noise similar to white noise.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL could make it easy for someone to generate random noise and customize it
+to their liking.
+It is easy to find white, pink, brown, blue, violet, and grey noise but it is
+harder to make noise from a different frequency spectrum.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I used to enjoy listening to random noise in order to fall asleep.
+Now that I know more math and cs, I thought about writing a program to generate
+random noise.
+I've also been having trouble sleeping because I've been sick, so I thought
+about random noise again.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Generate random noise for relaxation.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The interface should be graphical. Users can draw the top of a frequency 
+spectrum like those displayed 
+[here](https://en.wikipedia.org/wiki/Colors_of_noise).
+
+Users can also utilize a slider to change left-right balance for stereo volume.
+
+The graphical interface is simple to use and hopefully intuitive.
+People who are interested in random noise also use colors to name noise,
+so a graphical interface may be helpful for generating more complex noise.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would use the currently displayed frequency spectrum to generate
+noise.
+The noise should stay updated as the user modifies the spectrum.
+So the noise should change with the drawing without interruptions.
+
+If an error occurs, the noise should stop as really terrible sounds could
+result from the error. Then an error message should appear on the graphical
+interface.
+
+A possible error could be a strange spectrum that is discontinuous.
 
```

Comment:
Are you thinking of letting the user stop the random noise within a loop by creating discontinuities in the spectrum.